modules/ocf/facts.d/le-cert-info

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+import json
+import os
+from datetime import datetime
+
+from cryptography import x509
+from cryptography.hazmat.backends import default_backend
```

Comment:
I'm a little worried actually about using these, which I didn't notice the first time, since they are actually from the third-party `python3-cryptography` package. I think this script just won't run on a bootstrap of a new system until `python3-cryptography` is installed as a dependency of [`python3-paramiko` from `ocf::packages`](https://github.com/ocf/puppet/blob/487324734f4d34d5ae68cb79918ac0455cddc1d7/modules/ocf/manifests/packages.pp#L115).

Anyway, I think it should be ok since it's ok if facts fail to run, but maybe wrap this in a `try` block and if you get an `ImportError`, then just return `{}` from this fact or maybe some better fallback values if you can?

Reply:
I feel like if this depends on `python3-cryptography`, than we should add that explicit dependency in a letsencrypt manifest as well in case the `python3-paramiko` dependency gets removed later.

Will add the `try` block.